JaQuAD.ipynb

```diff
@@ -482,11 +483,12 @@
         "\n",
         "            ctx_start = tokens.index(self.tokenizer.sep_token_id) + 1\n",
         "            answer_start_index = ctx_start\n",
-        "            answer_end_index = len(offsets) - 1\n",
-        "            while offsets[answer_start_index][0] < start_char:\n",
+        "            while offsets[answer_start_index][1] < start_char:\n",
         "                answer_start_index += 1\n",
-        "            while offsets[answer_end_index][1] > start_char + len(answer):\n",
-        "                answer_end_index -= 1\n",
```

Comment:
We will get a smaller index number by 1 when the following token length is >= 2.